packages/react/src/components/Authenticator/FederatedSignIn/FederatedSignInButtons/__tests__/__snapshots__/FederatedSignInButton.test.tsx.snap

```diff
@@ -0,0 +1,69 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FederatedSignInButton renders as expected with Facebook provider 1`] = `
+<div>
+  <button
+    class="amplify-button amplify-field-group__control federated-sign-in-button"
+    data-fullwidth="false"
+    style="font-weight: var(--amplify-font-weights-normal); gap: 1rem;"
+    type="button"
+  >
+    <span
+      class="amplify-text"
+    >
+      Sign in with Facebook
+    </span>
+  </button>
+</div>
+`;
+
+exports[`FederatedSignInButton renders as expected with Google provider 1`] = `
+<div>
+  <button
+    class="amplify-button amplify-field-group__control federated-sign-in-button"
+    data-fullwidth="false"
+    style="font-weight: var(--amplify-font-weights-normal); gap: 1rem;"
+    type="button"
+  >
+    <span
+      class="amplify-text"
+    >
+      Sign in with Google
+    </span>
+  </button>
+</div>
+`;
+
+exports[`FederatedSignInButton renders as expected with LoginWithAmazon provider 1`] = `
+<div>
+  <button
+    class="amplify-button amplify-field-group__control federated-sign-in-button"
+    data-fullwidth="false"
+    style="font-weight: var(--amplify-font-weights-normal); gap: 1rem;"
+    type="button"
+  >
+    <span
+      class="amplify-text"
+    >
+      Sign in with LoginWithAmazon
```

Comment:
Is this what we actually display?

Reply:
Oh good point, nope, `FederatedSignInButton` takes in additional  `text` parameter to customize this. Let me use it to make it render what we actually use

Reply:
Done, [50a1c31](https://github.com/aws-amplify/amplify-ui/pull/2706/commits/50a1c3196656f2ddfc0e8d4b75ab572ff6b659ca)